src/components/KEmptyState/KEmptyState.vue

```diff
@@ -1,143 +1,167 @@
 <template>
-  <section
-    class="empty-state-wrapper"
-    :class="{ 'is-error': isError }"
+  <div
+    class="k-empty-state"
+    :class="[iconVariant]"
   >
-    <div class="empty-state-title">
-      <div
-        v-if="isError || icon"
-        class="k-empty-state-icon card-icon"
-        :class="{ 'warning-icon': isError }"
-      >
-        <KIcon
-          :color="isError ? iconColor || `var(--kui-color-text-neutral-strongest, ${KUI_COLOR_TEXT_NEUTRAL_STRONGEST})` : iconColor"
-          :icon="icon ? icon : 'warning'"
-          :secondary-color="isError ? iconSecondaryColor || 'currentColor' : iconSecondaryColor"
-          :size="iconSize"
-        />
-      </div>
-      <div
-        v-if="$slots.title"
-        class="k-empty-state-title-header"
-      >
-        <slot name="title" />
-      </div>
-    </div>
     <div class="empty-state-content">
-      <div
-        v-if="$slots.message"
-        class="k-empty-state-message"
-      >
-        <slot name="message" />
+      <div class="empty-state-icon">
+        <slot name="icon">
+          <component
+            :is="getEmptyStateIcon"
+            :color="getIconColor"
+            :size="KUI_ICON_SIZE_60"
+          />
+        </slot>
       </div>
+      <span
+        v-if="title"
+        class="empty-state-title"
+      >
+        {{ title }}
+      </span>
```

Comment:
Oh soz, I missed this in the previous PR

Q: How much do we expect people might what to add a `h*` in here for the title/header of the EmptyState? i.e. should the title be a slot?

Reply:
PR adding the title slot back: https://github.com/Kong/kongponents/pull/2053